docs/tutorials/shared_utilities/driver_tutorial.jl

```diff
@@ -139,3 +139,31 @@ updatefunc = update_radiation!;
 cb = ClimaLand.DriverUpdateCallback(updateat, updatefunc);
 
 # This callback must then be provided to the simulation [`solve`](https://docs.sciml.ai/DiffEqCallbacks/stable/) function.
+
+# # Using ERA5 data
+# If you wish to force your ClimaLand simulation with ERA5 reanalysis data, there is a helper function
+# which can make this easier than specifying each atmospheric variable as individual `TimeVaryingInput`
+# objects, and then making the `PrescribedAtmosphere` struct.
+# You first need a local path to the netcdf file with the ERA5 data. This file must have the following variables:
+# - "tp" Total precipitation as a mass/m^2, accumulated over an hour
```

Comment:
it is okay as is but this is equivalent to mass/m^2/hour, right? 
I know it is common to report precip like that, but is it not clearer to just give the full unit? just a thought.

Reply:
I updated this! Thanks